Mathlib/Topology/Homeomorph.lean

```diff
@@ -437,253 +438,253 @@ def homeomorphOfContinuousOpen (e : α ≃ β) (h₁ : Continuous e) (h₂ : IsO
   rfl
 #align homeomorph.Pi_congr_right_symm Homeomorph.piCongrRight_symm
 
 /-- `Equiv.piCongr` as a homeomorphism: this is the natural homeomorphism
-`Π i₁, β₁ i ≃ₜ Π i₂, β₂ i₂` obtained from a bijection `ι₁ ≃ ι₂` and homeomorphisms
-`β₁ i₁ ≃ₜ β₂ (e i₁)` for each `i₁ : ι₁`. -/
+`Π i₁, Y₁ i ≃ₜ Π i₂, Y₂ i₂` obtained from a bijection `ι₁ ≃ ι₂` and homeomorphisms
+`Y₁ i₁ ≃ₜ Y₂ (e i₁)` for each `i₁ : ι₁`. -/
 @[simps! apply toEquiv]
-def piCongr {ι₁ ι₂ : Type*} {β₁ : ι₁ → Type*} {β₂ : ι₂ → Type*}
-    [∀ i₁, TopologicalSpace (β₁ i₁)] [∀ i₂, TopologicalSpace (β₂ i₂)]
-    (e : ι₁ ≃ ι₂) (F : ∀ i₁, β₁ i₁ ≃ₜ β₂ (e i₁)) : (∀ i₁, β₁ i₁) ≃ₜ ∀ i₂, β₂ i₂ :=
+def piCongr {ι₁ ι₂ : Type*} {Y₁ : ι₁ → Type*} {Y₂ : ι₂ → Type*}
+    [∀ i₁, TopologicalSpace (Y₁ i₁)] [∀ i₂, TopologicalSpace (Y₂ i₂)]
+    (e : ι₁ ≃ ι₂) (F : ∀ i₁, Y₁ i₁ ≃ₜ Y₂ (e i₁)) : (∀ i₁, Y₁ i₁) ≃ₜ ∀ i₂, Y₂ i₂ :=
   (Homeomorph.piCongrRight F).trans (Homeomorph.piCongrLeft e)
 
 -- porting note: TODO: align the order of universes with `Equiv.ulift`
-/-- `ULift α` is homeomorphic to `α`. -/
-def ulift.{u, v} {α : Type u} [TopologicalSpace α] : ULift.{v, u} α ≃ₜ α where
+/-- `ULift X` is homeomorphic to `X`. -/
+def ulift.{u, v} {X : Type u} [TopologicalSpace X] : ULift.{v, u} X ≃ₜ X where
   continuous_toFun := continuous_uLift_down
   continuous_invFun := continuous_uLift_up
   toEquiv := Equiv.ulift
 #align homeomorph.ulift Homeomorph.ulift
 
 section Distrib
 
-/-- `(α ⊕ β) × γ` is homeomorphic to `α × γ ⊕ β × γ`. -/
-def sumProdDistrib : Sum α β × γ ≃ₜ Sum (α × γ) (β × γ) :=
+/-- `(X ⊕ Y) × Z` is homeomorphic to `X × Z ⊕ Y × Z`. -/
+def sumProdDistrib : Sum X Y × Z ≃ₜ Sum (X × Z) (Y × Z) :=
   Homeomorph.symm <|
-    homeomorphOfContinuousOpen (Equiv.sumProdDistrib α β γ).symm
+    homeomorphOfContinuousOpen (Equiv.sumProdDistrib X Y Z).symm
         ((continuous_inl.prod_map continuous_id).sum_elim
           (continuous_inr.prod_map continuous_id)) <|
       (isOpenMap_inl.prod IsOpenMap.id).sum_elim (isOpenMap_inr.prod IsOpenMap.id)
 #align homeomorph.sum_prod_distrib Homeomorph.sumProdDistrib
 
-/-- `α × (β ⊕ γ)` is homeomorphic to `α × β ⊕ α × γ`. -/
-def prodSumDistrib : α × Sum β γ ≃ₜ Sum (α × β) (α × γ) :=
+/-- `X × (Y ⊕ Z)` is homeomorphic to `X × Y ⊕ X × Z`. -/
+def prodSumDistrib : X × Sum Y Z ≃ₜ Sum (X × Y) (X × Z) :=
   (prodComm _ _).trans <| sumProdDistrib.trans <| sumCongr (prodComm _ _) (prodComm _ _)
 #align homeomorph.prod_sum_distrib Homeomorph.prodSumDistrib
 
 variable {ι : Type*} {σ : ι → Type*} [∀ i, TopologicalSpace (σ i)]
 
-/-- `(Σ i, σ i) × β` is homeomorphic to `Σ i, (σ i × β)`. -/
-def sigmaProdDistrib : (Σi, σ i) × β ≃ₜ Σi, σ i × β :=
+/-- `(Σ i, σ i) × Y` is homeomorphic to `Σ i, (σ i × Y)`. -/
+def sigmaProdDistrib : (Σi, σ i) × Y ≃ₜ Σi, σ i × Y :=
   Homeomorph.symm <|
-    homeomorphOfContinuousOpen (Equiv.sigmaProdDistrib σ β).symm
+    homeomorphOfContinuousOpen (Equiv.sigmaProdDistrib σ Y).symm
```

Comment:
I think these are the only Greek letters you missed. Would be reasonable to change to X.